iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/path/PathPatternTree.java

```diff
@@ -169,11 +172,12 @@ public boolean isEmpty() {
         && (pathPatternList == null || pathPatternList.isEmpty());
   }
 
-  public List<String> getAllDevicePatterns() {
+  public List<IDeviceID> getAllDevicePatterns() {
```

Comment:
need to talk about it, if we are sure that we need IDeviceId, we should use the way `List<PartialPath> getAllDevicePaths()`

Reply:
Fixed